### PR TITLE
Conflicting messages when a session is closed #4666

### LINKED ISSUE #4666: Conflicting messages when a session is closed
Received this from a student
image

### PR DESCRIPTION
**Pull request by Sumei1009**
fixes #4666

### CODE REVIEW

**Comment by wkurniawan07** [comment]
Looks good :+1:

File: src/main/java/teammates/common/util/Const.java [hunk 1]
```diff
@@ -1074,11 +1074,11 @@
                 + "However, you do not have access to view this student's profile";
         
         public static final String UNREGISTERED_STUDENT = "You are submitting feedback as <span class='text-danger text-bold text-large'>%s</span>. " 
-                + "You may submit feedback and view results without logging in. "
+                + "You may submit feedback for sessions that are currently open and view results without logging in. "
                 + "To access other features you need <a href='%s' class='link'>to login using a google account</a> "
                 + "(recommended).";
         public static final String UNREGISTERED_STUDENT_RESULTS = "You are viewing feedback results as <span class='text-danger text-bold text-large'>%s</span>. " 
-                + "You may submit feedback and view results without logging in. "
+                + "You may submit feedback for sessions that are currently open and view results without logging in. "
```
Comment (damithc) [inline src/main/java/teammates/common/util/Const.java]: The two messages seem to have a common segment. Perhaps we can get rid of the duplication at the same time?

Comment (damithc) [inline src/main/java/teammates/common/util/Const.java]: `google` --> `Google`

**Comment by damithc** [comment]
Minor change requested.

**Comment by Sumei1009** [comment]
I have extract the common part into a constant

File: src/main/java/teammates/common/util/Const.java [hunk 2]
```diff
@@ -1073,14 +1073,13 @@
...
-                + "You may submit feedback and view results without logging in. "
-                + "To access other features you need <a href='%s' class='link'>to login using a google account</a> "
-                + "(recommended).";
+                + UNREGISTERED_STUDENT_MESSAGE;
         public static final String UNREGISTERED_STUDENT_RESULTS = "You are viewing feedback results as <span class='text-danger text-bold text-large'>%s</span>. " 
-                + "You may submit feedback and view results without logging in. "
-                + "To access other features you need <a href='%s' class='link'>to login using a google account</a> "
-                + "(recommended).";
+                + UNREGISTERED_STUDENT_MESSAGE;
     }
```
Comment (damithc) [inline src/main/java/teammates/common/util/Const.java]: While we are at it, can you fix the line length too. Some of these lines are too long.

**Comment by damithc** [comment]
Another small cosmetic change requested.

**Comment by wkurniawan07** [comment]
@Sumei1009, the merge commit message is wrong. But what worried me more is: have you tested before merging?

**Comment by Sumei1009** [comment]
I remembered all the tests were passed. What's wrong with the merge?

**Comment by wkurniawan07** [comment]
Your merge commit message is "Merge pull request ..." instead of "[1234] Issue title". I presume you used GitHub's merging tool?
No worries about the tests, more commits have been made to master branch since then and all is well.

**Pull request merged**